docker.py

```diff
@@ -521,6 +522,9 @@ def make_dockerfile(conf):
             'RUN pip install -U %s && rm -rf ~/.cache/pip\n' %
             ' '.join(['"%s"' % req for req in requires]))
 
+        if scipy is not None:
```

Comment:
Write a comment why it is required.

Reply:
Please write detailed description. For example:

SciPy depends on C-API interface of NumPy. When you install different version of NumPy, it breaks compatibility and causes an error. So you need to install SciPy from its source to link NumPy you use.

Reply:
Thanks!